production/helm/templates/promtail/daemonset.yaml

```diff
@@ -57,10 +57,14 @@ spec:
               name: http-metrics
           securityContext:
             {{- toYaml .Values.promtail.securityContext | nindent 12 }}
+          {{- if .Values.promtail.livenessProbe }}
```

Comment:
Let's just remove the probes. There are no probes defined and promtail doesn't have a health check endpoint. WDYT @davkal ?

Reply:
I went ahead and removed this.. @davkal  If you disagree i can add it back.